linux-tools/gawk/gawk.sh

```diff
@@ -114,7 +114,11 @@ function runtest()
 	tc_register "$tst"
 
 	srcdir=. PATH_SEPARATOR=: make -f Makefile.am $tst >$stdout 2>$stderr
-	tc_pass_or_fail $?
+	RC=$?
+        if [ $tst = "fnamedat" ] || [ $tst = "fnarray" ] || [ $tst = "fnarray2" ] || [ $tst = "fnarydel" ] || [ $tst = "fnasgnm" ] || [ $tst = "fnparydl" ] || [ $tst = "funsmnam" ] || [ $tst = "gsubasgn" ] || [ $tst = "parseme" ] || [ $tst = "badargs" ] || [ $tst = "match2" ] && [ $RC -eq 0 ]; then
+                tc_ignore_warnings "make:"
+        fi
```

Comment:
tc_ignore_warnings "make:" and tc_ignore_warnings "ar:"   
This will result in ignoring anything with "make/ar" ( which may be an error and we may miss ) in future. So makes sense to use proper warning messages instead of using just "make" or "ar" .